package.json

```diff
@@ -5,14 +5,14 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
-    "gatsby": "^4.0.0-next.1",
-    "gatsby-plugin-image": "^2.0.0-next.1",
-    "gatsby-plugin-manifest": "^4.0.0-next.1",
-    "gatsby-plugin-offline": "^5.0.0-next.1",
-    "gatsby-plugin-react-helmet": "^5.0.0-next.1",
-    "gatsby-plugin-sharp": "^4.0.0-next.1",
-    "gatsby-source-filesystem": "^4.0.0-next.1",
-    "gatsby-transformer-sharp": "^4.0.0-next.1",
+    "gatsby": "^4.0.0-zz-next.1",
```

Comment:
I would expect to see the `package-lock.json` updated as well. Did you use `npm` or `yarn`?

Reply:
yarn

Reply:
i saw that it had a yarn.lock file so i used yarn and these were the only files that were added when i did `git add .`

Reply:
Since this has both yarn and npm lock files. I'd recommend using the `npm i` CLI then `yarn import` (if needed)